<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAsignarRol</class>
 <widget class="QMainWindow" name="frmAsignarRol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignar rol a usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Usuario:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rol:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbxUsuario"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbxRol"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnAsignarRol">
       <property name="text">
        <string>Asignar Rol</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>169</y>
      <width>471</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Id Usuario Rol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Apellido</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre Usuario</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rol</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>169</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnEliminarRolAsignado">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
